common/app/conf/switches/ABTestSwitches.scala

```diff
@@ -64,4 +64,13 @@ trait ABTestSwitches {
     sellByDate = Some(LocalDate.of(2024, 1, 31)),
     exposeClientSide = true,
   )
+   Switch(
+    ABTests,
+    "ab-crosswords-mobile-banner",
+    "Test mobile banner in crossword pages.",
+    owners = Seq(Owner.withEmail("commercial.dev@theguardian.com")),
+    safeState = Off,
+    sellByDate = Some(LocalDate.of(2024, 1, 31)),
```

Comment:
Extend this

Reply:
yes of course, thought of 25+ days from start, should be Feb